packages/moonstone/ToggleIconItem/ToggleIconItem.js

```diff
@@ -0,0 +1,119 @@
+import kind from '@enact/core/kind';
+import React from 'react';
+import PropTypes from 'prop-types';
+import EnactPropTypes from '@enact/core/internal/prop-types';
+import Icon from '../Icon';
+import componentCss from './ToggleIconItem.module.less';
+
+/**
+ * Renders an `ToggleItem` with a customized icon.
+ *
+ * @class ToggleIconItem
+ * @memberof moonstone/ToggleIconItem
+ * @ui
+ * @public
+ */
+
+// eslint-disable-next-line
+const itemIconCreator = (position) => ({css, itemIcon, itemIconPosition}) => {
+	if (position === itemIconPosition) {
+		return (
+			itemIcon !== null ?
```

Comment:
We could move this up to the `if` 2 lines above and have a simpler return:
```if (position === itemIconPosition && itemIcon != null) {```
Also, `!= null` will catch `undefined` as well as `null`, rather than `!== null` which only catches `null`.